ansible/roles/etcd/tasks/main.yml

```diff
@@ -1,4 +1,18 @@
 ---
+- name: ensure firewalld is installed
+  yum:
```

Comment:
Does adding it to the `common_rpms` not get it installed on the nodes?

Reply:
it doesn't get added to the etcd nodes in the common_rpms. re-working it somehow so it can be a common rpm across all nodes would be ideal.

Reply:
I agree with @alexbrand - I think this can all go in `common`

Reply:
Looks like the `etcd` play does not run the `common` role. I'll defer to @craigtracey on whether it should or not.

Reply:
Then one of you should probably test this, because it never worked for me. It only installed it on the k8s nodes and _not_ on etcd.

Reply:
Updated code now so it is working from common and etcd hosts use the `common` role